openff/interchange/components/interchange.py

```diff
@@ -367,6 +385,59 @@ def to_openmm_topology(
             ensure_unique_atom_names=ensure_unique_atom_names,
         )
 
+    def to_openmm_simulation(
+        self,
+        integrator: "openmm.Integrator",
+        combine_nonbonded_forces: bool = True,
+        add_constrained_forces: bool = False,
+        **kwargs,
+    ) -> "openmm.app.Simulation":
+        """
+        Export this Interchange to an OpenMM `Simulation` object.
+
+        Positions are set on the `Simulation` if present on the `Interchange`.
+
+        Parameters
+        ----------
+        integrator : subclass of openmm.Integrator
+            The integrator to use for the simulation.
+        combine_nonbonded_forces : bool, default=False
+            If True, an attempt will be made to combine all non-bonded interactions into a single
+            openmm.NonbondedForce.
+            If False, non-bonded interactions will be split across multiple forces.
+        add_constrained_forces : bool, default=False,
+            If True, add valence forces that might be overridden by constraints, i.e. call `addBond` or `addAngle`
+            on a bond or angle that is fully constrained.
+
+        Returns
+        -------
+        simulation : openmm.app.Simulation
+            The OpenMM simulation object, possibly with positions set.
+
+        """
```

Comment:
```suggestion
        Returns
        -------
        simulation : openmm.app.Simulation
            The OpenMM simulation object, possibly with positions set.

        Examples
        --------
        
        Create an OpenMM simulation with a Langevin integrator:
        
        >>> import openmm
        >>>
        >>> simulation = interchange.to_openmm_system(
        ...     openmm.LangevinMiddleIntegrator(
        ...         293.15 * openmm.unit.kelvin,
        ...         1.0 / openmm.unit.picosecond,
        ...         2.0 * openmm.unit.femtosecond,
        ...     )
        ... )
        
        Add a barostat:
        
        >>> simulation.system.addForce(
        ...     openmm.MonteCarloBarostat(
        ...         1.00 * openmm.unit.bar, 
        ...         293.15 * openmm.unit.kelvin, 
        ...         25,
        ...     )
        ... )
        
        """
```

If we stick with the current signature, it would be nice to include examples here.

Reply:
My favorite docexamples are the ones that other people write